packages/slonik/src/__test__/helpers/utils.ts

```diff
@@ -14,6 +14,68 @@ const getFilterDataset = () => {
     { key: "name", operator: "in", value: "Test1, Test2" },
     { key: "id", operator: "bt", value: "10, 20" },
     { key: "id", not: true, operator: "bt", value: "10, 20" },
+    { AND: [{ key: "id", operator: "lt", value: 10 }] },
```

Comment:
This is not valid filter syntax.
`AND` and `OR` filter operator should have at-least 2 value. 
Structure
```
{
   AND: [{key.....}, {key......}]
}
```